### PR TITLE
Make login for all roles

### PR DESCRIPTION
**Pull request by Wallert**
Make a login for all roles.

### CODE REVIEW

**Review by Sarhsian — COMMENTED** [review]
Good

**Review by MishkaBaloo — COMMENTED** [review]
Good work

File: EasyRestTests/PageObjects/SignInPageObject.cs [hunk 1]
```diff
@@ -34,6 +34,34 @@ public void SignIn(string email, string password)
             SendTextToPasswordTextField(password);
             ClickSubmitButton();
         }
+        public void SignInAsClient()
```
Comment (dazeforlife) [inline EasyRestTests/PageObjects/SignInPageObject.cs:37]: Nice!

**Comment by w1nyx** [comment]
Merging!

**Review by vadymhrnk — COMMENTED** [review]
Great!

**Pull request merged**